src/concrete_solve.jl

```diff
@@ -354,14 +361,15 @@ function DiffEqBase._concrete_solve_adjoint(
         sol = solve(_prob, alg, args...; save_noise = true, save_start = true,
             save_end = true, kwargs_fwd...)
     end
+    time = current_time(sol)
 
     # Force `save_start` and `save_end` in the forward pass This forces the
     # solver to do the backsolve all the way back to `u0` Since the start aliases
     # `_prob.u0`, this doesn't actually use more memory But it cleans up the
     # implementation and makes `save_start` and `save_end` arg safe.
     if sensealg isa BacksolveAdjoint
         # Saving behavior unchanged
-        ts = sol.t
+        ts = time
```

Comment:
Why change the standard naming? Naming an array `time` instead of a scalar value is weird and not grammatically correct...

Reply:
I can change the naming or potentially if there's a better solution, can implement that

Reply:
That's fine, it could still be called `ts`. If you want to write it, `times`. But I don't understand why syntactic changes are mixed with a feature PR.

Reply:
This should be fixed now with https://github.com/SciML/SciMLSensitivity.jl/pull/1057/commits/6952e1b8d3a738c5301e1514a53f193d367243d0 and https://github.com/SciML/SciMLSensitivity.jl/pull/1057/commits/95cad8587ea1d34889cb625bb104e0063e1fb385